triage.md

```diff
@@ -0,0 +1,68 @@
+    - Try to assign owners if it is a high priority beaking change
+
+- [Bugs](https://github.com/opensearch-project/opensearch-benchmark/issues?q=is%3Aissue+is%3Aopen+label%3Abug)
+    - Search for Issues which are labeled as bugs
+    - Comment with a status update such as In Progress, Not Planned, Needs more information
+
+- [Enhancements](https://github.com/opensearch-project/opensearch-benchmark/issues?q=is%3Aissue+is%3Aopen+sort%3Acreated-desc+label%3Aenhancement+)
+    - Search for Issues which are feature requests/enhancements
+    - Comment with a status update and assign owners if it is going to be the next release candidate
+
+- [Documentation](https://github.com/opensearch-project/opensearch-benchmark/issues?q=is%3Aopen+is%3Aissue+label%3Adocumentation)
+    - These Issues although not breaking are important for better customer experience
+    - Feel free to encourage the requester to raise a PR, these can be tagged as good first Issue as well
+
+### Steps to triage [opensearch-benchmark-workloads](https://github.com/opensearch-project/opensearch-benchmark-workloads)
+- [Untriaged](https://github.com/opensearch-project/opensearch-benchmark-workloads/issues?q=is%3Aopen+is%3Aissue+no%3Alabel)
+    - First check if any of the untriaged issue is a duplicate. If yes, then comment the duplicate Issue link and close the duplicate
+    - Assign appropriate labels for these Issues such as bug, enhancements, breaking etc.
+    - If possible assign owners who could be [SME](https://en.wikipedia.org/wiki/Subject-matter_expert) for the Issue or volunteers if any
+
+- [Breaking](https://github.com/opensearch-project/opensearch-benchmark-workloads/issues?q=is%3Aopen+is%3Aissue+label%3Abreaking)
+    - Search for Issues which are labeled as breaking
+    - Try to assign owners if it is a high priority beaking change
+
+- [Bugs](https://github.com/opensearch-project/opensearch-benchmark-workloads/issues?q=is%3Aopen+is%3Aissue+label%3Abug)
+    - Search for Issues which are labeled as bugs
+    - Comment with a status update such as In Progress, Not Planned, Needs more information
+
+- [Enhancement](https://github.com/opensearch-project/opensearch-benchmark-workloads/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement)
+    - Search for Issues which are feature requests/enhancements
+    - Comment with a status update and assign owners if it is going to be the next release candidate
+
+- [Documentation](https://github.com/opensearch-project/opensearch-benchmark-workloads/issues?q=is%3Aopen+is%3Aissue+label%3Adocumentation)
+    - These Issues although not breaking are important for better customer experience
+    - Feel free to encourage the requester to raise a PR, these can be tagged as good first Issue as well
+
+*There could be an overlap of Issues in more than 1 category such as Breaking and Bugs or Breaking and High Priority. In this case try to triage a different Issue which is not already covered*
```

Comment:
Notwithstanding the steps laid out above, some issues may fall into multiple categories and need to be triaged only once.